register.lic

```diff
@@ -0,0 +1,76 @@
+=begin
+  Search a deed register for the specified crafting material.
```

Comment:
Will you be adding documentation for this script to elanthipedia?

Reply:
Yes.

Reply:
Excellent!

Could you add a link to the header like `Documentation: https://elanthipedia.play.net/Lich_script_repository#register`?